website/content/en/preview/upgrading/compatibility.md

```diff
@@ -15,9 +15,9 @@ Before you begin upgrading Karpenter, consider Karpenter compatibility issues re
 
 [comment]: <> (the content below is generated from hack/docs/compataiblitymetrix_gen_docs.go)
 
-| KUBERNETES |       1.24        |   1.25   |   1.26   |   1.27   |   1.28   |   1.29   |   1.30   |
-|------------|-------------------|----------|----------|----------|----------|----------|----------|
-| karpenter  | \>= 0.21 \<= 0.37 | \>= 0.25 | \>= 0.28 | \>= 0.28 | \>= 0.31 | \>= 0.34 | \>= 0.37 |
+| KUBERNETES |   1.25   |   1.26   |   1.27   |   1.28   |   1.29   |   1.30   | 1.31  |
+|------------|----------|----------|----------|----------|----------|----------|-------|
+| karpenter  | \>= 0.25 | \>= 0.28 | \>= 0.28 | \>= 0.31 | \>= 0.34 | \>= 0.37 | 1.0.x |
```

Comment:
While this is correct, we should probably say >= 1.0.5. Any version before that should work, but it will report an error that the k8s version is unsupported.

Reply:
Done.